### DISCUSSION TITLE
Possible to increase height of error dialog?

### DISCUSSION THREAD

**Discussion by heyitaki** [yoavbls/pretty-ts-errors#57] [Q&A] [answered]
For larger types, the resulting message is formatted such that the vertical height of the error message increases drastically (naturally). I'm wondering if there is some VSCode setting or hack that will let me increase the height of this error box to better view the types instead of having to scroll up and down constantly. 

I know that VSCode implemented resizable suggestions in 2020, I may be missing something if this is extensible to the error dialog as well.

**Comment by yoavbls** [comment]
Hey @heyitaki!
Currently, it isn't possible, but hovers on VSCode is going to be resizable very soon!
We're waiting for this PR to be merged:
https://github.com/microsoft/vscode/pull/176744

**Comment by ecker00** [comment]
Or a way to view the errors in their own window, as investigating larger types can be quite tricky in the constrained space.

**Comment by yoavbls** [comment]
Resizable hovers get in in the latest VSCode release 🙂

**Comment by yoavbls** [comment]
@ecker00 I agree, @typeholes made a POC for such a view a long time ago, It's not so simple because
WebViews work differently than hovers in VSCode but it's a good idea and I will appreciate any help in that

**Comment by ErikDakoda** [comment]
Since the VSCode PR https://github.com/microsoft/vscode/issues/14165 was merged, how come I still can't resize the height of the error dialog?

**Reply by yoavbls** [reply]
@ErikDakoda Can't you drag the corner of the hover window?
Keep notice that if the hover is below only the right-bottom corner is draggable
and if it's above the only draggable hover is the right-top corner.

![CleanShot 2024-01-13 at 18 53 30@2x](https://github.com/yoavbls/pretty-ts-errors/assets/5693018/8de0a765-879b-4f0c-b08d-4d39689fa722)

**Comment by kevinramharak** [comment, marked as answer]
Closing this as this feature was merged in https://github.com/microsoft/vscode/pull/178811
See VS Code update https://code.visualstudio.com/updates/v1_80#_resizable-content-hover
